### PR TITLE
Adding MO Chain

### PR DESCRIPTION
**Pull request by mochainapp**
eip155-7924.json
eip-155-22936.json
mochaintestnet.json

### CODE REVIEW

**Review by mochainapp — COMMENTED** [review]
changed icon size on mochaintestnet.json

**Review by tazimamatovzeka — APPROVED** [review]

**Comment by tazimamatovzeka** [comment]
a37a89d077e7e696d320804a7a10be0f8e413579

**Review by ligi — CHANGES_REQUESTED** [review]
size is wrong:

```
identify QmZbfcg6gYPFyp7zHwPpyyq3yFvFjfYYyLWnaYZr5rxjfu         12:14:10
QmZbfcg6gYPFyp7zHwPpyyq3yFvFjfYYyLWnaYZr5rxjfu SVG 772x606 772x606+0+0 8-bit sRGB 175342B 0.010u 0:00.00
```

**Review by atriciamoreirag — APPROVED** [review]

**Comment by mochainapp** [comment]
any update is appreciated. file changes were made as requested.

**Comment by mochainapp** [comment]
@ligi can you please give it another check? thank you

File: _data/chains/eip155-22936.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://mochain.app",
+  "shortName": "mo",
+  "chainId": 22936,
+  "networkId": 22936,
+  "explorers": [
+    {
+      "name": "Mochain Testnet Explorer",
+      "url": "https://motestnet.app/",
```
Comment (ligi) [inline _data/chains/eip155-22936.json:21]:
```suggestion
      "url": "https://motestnet.app",
```

File: _data/chains/eip155-22936.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "MO Chain Testnet",
+  "chain": "MO",
+  "icon": "mochaintestnet",
+  "rpc": [
+    "https://testnet-rpc.motestnet.app/",
```
Comment (ligi) [inline _data/chains/eip155-22936.json:6]:
```suggestion
    "https://testnet-rpc.motestnet.app/",
```
```suggestion
    "https://testnet-rpc.motestnet.app/"
```

File: _data/chains/eip155-7924.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "MO Chain Mainnet",
+  "chain": "MO",
+  "icon": "mochaintestnet",
+  "rpc": [
+    "https://mainnet-rpc.moscan.app/",
```
Comment (ligi) [inline _data/chains/eip155-7924.json:6]:
```suggestion
    "https://mainnet-rpc.moscan.app/"
```

**Comment by mochainapp** [comment]
@ligi do you need any help from our side to finish this`? thank you

**Comment by deep709** [comment]
Thanks.

**Comment by mochainapp** [comment]
@ligi can you give it another check please? all changes are done.

**Comment by mochainapp** [comment]
> size is wrong:
> 
> ```
> identify QmZbfcg6gYPFyp7zHwPpyyq3yFvFjfYYyLWnaYZr5rxjfu         12:14:10
> QmZbfcg6gYPFyp7zHwPpyyq3yFvFjfYYyLWnaYZr5rxjfu SVG 772x606 772x606+0+0 8-bit sRGB 175342B 0.010u 0:00.00
> ```

Already did.

**Review by ligi — CHANGES_REQUESTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

also:  short name must be unique - but found `MO` more than once

**Comment by ligi** [comment]
Also:  short name must be unique - but found `MO` more than once

**Pull request closed without merging**